README.md

```diff
@@ -13,11 +13,31 @@ for querying role IDs from an OMERO database.
 
 Put your OMERO server configuration into `etc/omero.properties`, run
 
-    gradle run --args='etc/omero.properties'
+    gradle run --args=etc/omero.properties
 
 and try
 
-    src/scripts/fetch-ms-zarr.py -h
+    src/scripts/fetch-ms-zarr.py --endpoint-url http://localhost:8080/ --url-format '{url}image/{image}.zarr/' 1234
+
+
+### Configuration
+
+`etc/omero.properties` may include:
+
+`omero.ms.zarr.buffer-cache.size`
+: pixel buffer cache size, default 16
+
+`omero.ms.zarr.chunk.size.min`
+: minimum chunk size (not guaranteed), default 1048576; applies before compression
+
+`omero.ms.zarr.compress.zlib.level`
+: zlib compression level for chunks, default 6
+
+`omero.ms.zarr.net.path.image`
+: URI template path for getting image data, default `/image/{image}.zarr/` where `{image}` signifies the image ID and is mandatory
+
+`omero.ms.zarr.net.port`
+: the TCP port on which the HTTP server should listen, default 8080
```

Comment:
Since you're updating the README could you also mention the other properties that are most likely required (`omero.db.*`, `omeor.data.dir`)?

Reply:
I figured they're already covered by the statement above?

> Put your OMERO server configuration into `etc/omero.properties` ...

Reply:
I think it assumes too much. To me that could mean:
- properties to connect to OMERO (like `ice.config`)
- output of `omero config get` which may include random things like certificate paths, custom java properties for OMERO, LDAP connection info, etc.
- copy `OMERO.server/etc/omero.properties` to the microservice